wagtail/admin/templates/wagtailadmin/home/pages_for_moderation.html

```diff
@@ -1,75 +1,75 @@
 {% load i18n wagtailadmin_tags %}
 {% if page_revisions_for_moderation %}
-    <div class="panel nice-padding">{# TODO try moving these classes onto the section tag #}
-        <section>
-            <h2>{% trans 'Pages awaiting moderation' %}</h2>
-            <table class="listing">
-                <col />
-                <col width="30%"/>
-                <col width="15%"/>
-                <col width="15%"/>
-                <thead>
+<section class="object collapsible">
+    <h2 class="title-wrapper">{% trans 'Pages awaiting moderation' %}</h2>
+    <div class="object-layout">
+        <table class="listing">
```

Comment:
The large spacing between the title and the first table item (with the dotted line) also looks a bit odd/like an additional item to me - would it be possible to try it with a smaller one?

Reply:
This needs design input as the request came from that direction via Dex

Reply:
Eh, if it's part of the design, ignore me on this one.